client/src/main/scala/Client.scala

```diff
@@ -0,0 +1,9 @@
+import scala.scalajs.js.JSApp
+import org.scalajs.dom
+
+object Client extends JSApp {
+  def main(): Unit = {
+    dom.document.title = "Hello world!"
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal "Hello world!" appears 2 times in the file.](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594746741/issues/source?bid=3301982&fileBranchId=3306761#l6)